common/beerocks/scripts/CMakeLists.txt

```
@@ -1,4 +1,10 @@
+configure_file(
+        "${CMAKE_CURRENT_SOURCE_DIR}/prplmesh_utils.sh.in"
+        "${CMAKE_CURRENT_BINARY_DIR}/prplmesh_utils.sh"
+        )
```

Since this is a shell script which may use `${VAR}` syntax, add `@ONLY`:
```suggestion
        @ONLY 
        )
```

---

ok